ospfd/ospf_interface.c

```diff
@@ -1392,6 +1396,9 @@ static int ospf_ifp_up(struct interface *ifp)
 		ospf_if_up(oi);
 	}
 
+	if (HAS_LINK_PARAMS(ifp))
+		ospf_mpls_te_update_if(ifp);
+
```

Comment:
I understand the purpose of these 2 additions. But, do we really need it? Indeed, MPLS-TE is working fine currently without these 2 calls to `ospf_mpls_te_update()`

Reply:
MPLS-TE is working find for ospfd today because ospf_interface_link_params/ospf_mpls_te_update_if is called regardless of whether the link-params changed in the ZEBRA_INTERFACE_LINK_PARAMS message.

Once I added code to check for 'link-params changed', then this code change in ospf_interface.c was necessary.